tp-songs/01_folk/Asonance____Čarodějnice_z_Amesbury.tex

```diff
@@ -63,7 +63,7 @@
 \ks
 
 \zs
-Zuzana stála hrdě s hlavou hrdě vztyčenou
+Zuzana stála krásná s hlavou hrdě vztyčenou
```

Comment:
TP verze dává smyslu víc, ale zaprvé takhle to patří a za druhé by tam bylo dvakrát "hrdě".